docusaurus.config.js

```diff
@@ -116,6 +116,14 @@ const config = {
         disableSwitch: true,
         respectPrefersColorScheme: false,
       },
+
+      head: {
+        attributes: {
+          rel: 'stylesheet',
+          href: '<link href="https://fonts.cdnfonts.com/css/ibm-plex-sans" rel="stylesheet">',
```

Comment:
Shouldn't that be just the href value?

```suggestion
          href: 'https://fonts.cdnfonts.com/css/ibm-plex-sans',
```

Doc: https://docusaurus.io/docs/api/docusaurus-config#stylesheets